Content.Server/Body/Systems/InternalsSystem.cs

```diff
@@ -11,6 +11,9 @@
 using Content.Server.Popups;
 using Content.Server.DoAfter;
 using System.Threading;
+using Content.Shared.ActionBlocker;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Mobs.Systems;
```

Comment:
the 2 mob usings here are unused.

if you have an IDE, unused using will usually be greyed out.

If you do not, I recommend finding some tool for C# for whatever you prefer. Personally I use VS Code although I know there are independent plugins for lighter stuff like vim